.github/workflows/build.yml

```diff
@@ -230,19 +236,11 @@ jobs:
           if-no-files-found: error
 
       - name: Install Firebase app distribution plugin
-        if: ${{ github.ref_name == 'main' }}
+        if: ${{ inputs.distribute-to-firebase || github.event_name == 'push' }}
```

Comment:
We need to declare the push event trigger ☝️ 

```yaml
on:
  push:
    branches:
      - main
  workflow_dispatch: ...
```

Reply:
Ah, nice, thanks for noticing that, probably saved me 90 minutes of trying to figure out what was going on 😆 

f3a7e6fe92a61c8870b9798f15ada959b7d271a1